src/new.md

```diff
@@ -3,6 +3,48 @@ layout: page
+
+- [BinaryData]({{site.baseurl}}/software/api/reference/#T:Geotab.Checkmate.ObjectModel.BinaryData) (nuget only) - Fix issue deserializing enum values known to the server but unknown to older nuget package.
+
+- Calculated Engine Hours Search - As mentioned in 5.7.1904 What’s New, `DiagnosticEngineHoursAdjustmentId` is now interpolated using trips and `DiagnosticIgnitionId` values when a search includes a from/toDate value(s) to provide exact values by default.
+
+- [CompanyDetails]({{site.baseurl}}/software/api/reference/#T:Geotab.Checkmate.ObjectModel.Registration.CompanyDetails) - Add documentation describing field length limits. Added more specific error messages relating to max field lengths from `CreateDatabase` method.
+
+- [Device]({{site.baseurl}}/software/api/reference/#T:Geotab.Checkmate.ObjectModel.Device) - `HardwareId` is no longer returned as part of Device object. For more information regarding this change, please refer to this [community post](https://community.geotab.com/s/question/0D52J00007MIPRYSA5/sdk-notice-removal-of-device-property).
+
+- [DVIRDefect]({{site.baseurl}}/software/api/reference/#T:Geotab.Checkmate.ObjectModel.DVIRDefect) - Providing `RepairUser` and `RepairDateTime` are no longer supported for unrepaired `DVIRDefect`.
+
+- [DVIRLog]({{site.baseurl}}/software/api/reference/#T:Geotab.Checkmate.ObjectModel.DVIRLog) - `DefectList` must be provided with `DVIRLog`.
+
+- [GetFeed]({{site.baseurl}}/software/api/reference/#M:Geotab.Checkmate.Database.DataStore.GetFeed1) - Fixed corner case where it was possible to miss data in feed due to concurrency issue.
+
+- [GetFeed]({{site.baseurl}}/software/api/reference/#M:Geotab.Checkmate.Database.DataStore.GetFeed1) `StatusData` - Fix, providing a search to GetFeed `StatusData` containing a `DiagnosticSearch` which has no results within the provided limit of records will now return a feed version advanced by the results limit or remaining records when less then results limit.
+
+- [Get]({{site.baseurl}}/software/api/reference/#M:Geotab.Checkmate.Database.DataStore.Get1) `Diagnostic` - Fix issue searching by `DiagnosticType.ProprietaryFault` or `DiagnosticType.LegacyFault` causing error result.
+
+- [GoCurve]({{site.baseurl}}/software/api/reference/#T:Geotab.Checkmate.ObjectModel.GoCurve) - Added `IsIoxConnectionEnabled`. (Adds to `GO4v3`, `GO5`, `GO6`, `GO7`, `GO8`, `GO9`)
+
+- [Group]({{site.baseurl}}/software/api/reference/#T:Geotab.Checkmate.ObjectModel.Group) (nuget only) - Removed `left` and `right` parameters from constructor and `Group.Get` method.
+
+- [GroupRelations]({{site.baseurl}}/software/api/reference/#T:Geotab.Checkmate.ObjectModel.GroupRelations) - Added `AddInDatas` property. When `Group` linked `AddInData` (Feature Preview) is blocking `Group` remove a list blocking `AddInData` `Id`s will be retuend int the `GroupRelations` property of `GroupRelationViolatedException`.
```

Comment:
is blocking `Group` remove a list blocking `AddInData` `Id`s will be retuend int the `GroupRelations` property of `GroupRelationViolatedException`.

to 

is blocking `Group` a remove, a list of blocking `AddInData` `Id`s will be returned in the `GroupRelations` property of `GroupRelationViolatedException`.